### DISCUSSION TITLE
Turn the label chart into multi-channel

### DISCUSSION THREAD

**Discussion by LReion** [Project-MONAI/tutorials#1372] [Q&A] [answered]
I have a 3D label map that distinguishes organs by pixel values, and how I enhance the data to make it multichannel to facilitate the model to calculate losses

**Comment by KumoLiu** [comment, marked as answer]
Hi @LReion, I would suggest you use `AsDiscrete`.
```
out = AsDiscrete(to_onehot=num_classes)(data)
```
https://github.com/Project-MONAI/MONAI/blob/e0d0ce7d94fc63f3d32fdf03520048f6ae4acaab/monai/transforms/post/array.py#L131

Thanks!

**Reply by LReion** [reply]
Thank for you answer, have a good day.
